index.md

```diff
@@ -1,253 +1,95 @@
+  </p>
 </div>
 
 <div class="repo-group">
   <div class="repo-box">
-    <div class="repo-header">
-      <div class="repo-header-right">
-        <a href="https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/tx" alt="GitHub URL">
-          <i class="fa fa-github"></i>
-        </a>
-      </div>
-      @ethereumjs/tx
-    </div>
-    <div class="repo-description">
-      Implementation of the various transaction types.
-    </div>
-  </div>
-
-  <div class="repo-box">
-    <div class="repo-header">
-      <div class="repo-header-right">
-        <a href="https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/block" alt="GitHub URL">
-          <i class="fa fa-github"></i>
-        </a>
-      </div>
-      @ethereumjs/block
-    </div>
-    <div class="repo-description">
-      Block representations for all hardforks.
-    </div>
-  </div>
-
-  <div class="repo-box">
-    <div class="repo-header">
-      <div class="repo-header-right">
-        <a href="https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/blockchain" alt="GitHub URL">
-          <i class="fa fa-github"></i>
-        </a>
-      </div>
-      @ethereumjs/blockchain
-    </div>
-    <div class="repo-description">
-      Ethereum mainnet-compatible blockchain.
-    </div>
+    <img src="/assets/images/debug.png" alt="TypeScript EVM - Code Debugging">
   </div>
-  
 </div>
 
 <div class="separator"></div>
 
 <div class="repo-group">
-  <h3><i class="fa fa-cogs"></i> Protocol</h3>
-  <p>Implementations of protocol components and data structures.</p>
-</div>
+  <h3>Portal Network</h3>
 
-<div class="repo-group">
-  <div class="repo-box">
-    <div class="repo-header">
-      <div class="repo-header-right">
-        <a href="https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/trie" alt="GitHub URL">
-          <i class="fa fa-github"></i>
-        </a>
-      </div>
-      @ethereumjs/trie
-    </div>
-    <div class="repo-description">
-      Implementation of a Merkle Patricia Tree.
-    </div>
-  </div>
-
-  <div class="repo-box">
-    <div class="repo-header">
-      <div class="repo-header-right">
-        <a href="https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/rlp" alt="GitHub URL">
-          <i class="fa fa-github"></i>
-        </a>
-      </div>
-      @ethereumjs/rlp
-    </div>
-    <div class="repo-description">
-      RLP encoding and decoding.
-    </div>
-  </div>
+  <p>Beside we have our own TypeScript implementation - called <a href="https://github.com/ethereumjs/ultralight">Ultralight</a> -  for the <a href="https://github.com/ethereum/portal-network-specs">Portal Network</a>, which now gets its first production usage as a decentralized distributing system for historical block data to enable EIP-4444 history data expiry. We will continue to actively contribute to specification and testing for upcoming network types like the Portal state network. 
```

Comment:
```suggestion
  <p>Beside we have our own TypeScript implementation - called <a href="https://github.com/ethereumjs/ultralight">Ultralight</a> -  for the <a href="https://github.com/ethereum/portal-network-specs">Portal Network</a>, which now gets its first production usage as a decentralized distributing system for historical block data to enable EIP-4444 history data expiry. We will continue to actively contribute to specification and testing for upcoming network types like the Portal State network. 
```

Reply:
Will address these locally and not go over "Commit suggestion".